webpack.config.js

```diff
@@ -83,10 +83,14 @@ module.exports = [
 			filename: 'jp-search.bundle.js',
 		},
 		performance: isDevelopment
-			? {}
+			? {
+					maxAssetSize: 120000,
+					maxEntrypointSize: 120000,
+					hints: 'error',
```

Comment:
I have tried using warnings, but they are either completely silenced, or the `warning` setting doesn't work. Hence the `error`, so that it would be at least noticed.